Examples/AndroidStudio/app/build.gradle

```diff
@@ -31,10 +40,12 @@ android {
 }
 
 dependencies {
-    compile fileTree(dir: 'libs', include: ['*.jar'])
-
-    compile 'com.android.support:appcompat-v7:27.0.+'
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation 'com.android.support:appcompat-v7:27.+'
 
     // OneSignal SDK
-    compile 'com.onesignal:OneSignal:[3.8.1, 3.99.99)'
+    implementation 'com.onesignal:OneSignal:[3.9.2, 3.99.99)'
+
+    // Required for geotagging
+    implementation 'com.google.android.gms:play-services-location:15.0.1'
```

Comment:
Has this version been tested with other supported plugins that make use of GMS as well? For example, the location plugin?

Reply:
We haven't but the app with all supported plugins has no build issues, so this doesn't seem to be an issue at this time.